components/script/dom/htmlfieldsetelement.rs

```diff
@@ -52,9 +52,8 @@ impl HTMLFieldSetElementMethods for HTMLFieldSetElement {
         struct ElementsFilter;
         impl CollectionFilter for ElementsFilter {
             fn filter<'a>(&self, elem: &'a Element, _root: &'a Node) -> bool {
-                static TAG_NAMES: StaticStringVec = &["button", "fieldset", "input",
-                    "keygen", "object", "output", "select", "textarea"];
-                TAG_NAMES.iter().any(|&tag_name| tag_name == &**elem.local_name())
+                elem.downcast::<HTMLElement>()
```

Comment:
`Element` is the parent of `HTMLElement`, so we need to `downcast`.

https://html.spec.whatwg.org/multipage/dom.html#htmlelement